ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -534,6 +536,8 @@ export default class ConfirmTransactionBase extends Component {
       dismissOpenSeaToBlockaidBanner();
     };
 
+    const showTotals = Boolean(simulationData?.error);
```

Comment:
Are we displaying the total even if there's no balance changes?

Reply:
Yes, that was my understanding.